index.html

```diff
@@ -1,18 +1,18 @@
 <!DOCTYPE html>
-<html>
+<html lang="en">
   <head>
     <meta charset="UTF-8">
     <title>Hello World!</title>
   </head>
   <body>
     <h1>Hello World!</h1>
     <!-- All of the Node.js APIs are available in this renderer process. -->
-    We are using node <script>document.write(process.versions.node)</script>,
-    Chromium <script>document.write(process.versions.chrome)</script>,
-    and Electron <script>document.write(process.versions.electron)</script>.
+    We are using node <script type="text/javascript">document.write(process.versions.node)</script>,
```

Comment:
@akshayvinchurkar Electron applications always run in Chromium based environments though, where the default script type has been `text/javascript` for a few years.  In browser you might find this an issue (like IE6 era browsers) but as Electron is based on modern Chromium this tags are entirely optional